src/sentry/event_manager.py

```diff
@@ -907,31 +928,35 @@ def save(self, project_id, raw=False, assume_normalized=False):
             tagstore.delay_index_event_tags(
                 organization_id=project.organization_id,
                 project_id=project.id,
-                group_id=group.id,
+                group_id=group.id if group else None,
                 environment_id=environment.id,
                 event_id=event.id,
                 tags=event.tags,
                 date_added=event.datetime,
             )
 
         if event_user:
+            metrics_ = [
+                (tsdb.models.users_affected_by_project, project.id, (event_user.tag_value, )),
+            ]
+            if group:
```

Comment:
These metrics should be gone from redis TSDB entirely I think as they are computed from snuba, You can probably remove these calls altogether.

Reply:
yeah theres some other tagstore stuff in here that i assume also is going to get removed?